### DISCUSSION TITLE
Runtime error at startup - WinRT Unable to create wrapper DispatcherQueue

### DISCUSSION THREAD

**Discussion by najak3d** [dotnet/Microsoft.Maui.Graphics.Controls#97] [Q&A] [answered]
So I finally got Maui-check 0.9.0 to install and run.   (I had to manually delete folder:  "C:/users/{username}/.dotnet/tools/" before it would let me install it.   So with 0.9.0 synced, I can now compile and run the Sample.WinUI project.

But right at startup, it throws this error:
"ArgumentException: Unable to create a wrapper object.   WinRT object #### has type ...DispatcherQueue which cannot be assigned to ....DispatcherQueue"

The types in text look identical, so I'm guessing this might have to do with incompatibility with the runtime WinRT?   So as of yet, I cannot run the samples project, until I can resolve this runtime exception at startup:

![image](https://user-images.githubusercontent.com/10914515/145934943-720907fa-c66f-4b38-9be9-9da29d1e04a2.png)

Here is a snapshot of the SampleWinUI Dependencies:
![image](https://user-images.githubusercontent.com/10914515/145935277-a7df3ba8-79a8-4842-85c1-bddde7de8126.png)

How to resolve this?

**Comment by jsuarezruiz** [comment, marked as answer]
Should have been fixed by https://github.com/dotnet/Microsoft.Maui.Graphics.Controls/pull/101. Could you try again with the latest version?
